addons/account/static/src/components/document_file_uploader/document_file_uploader.js

```diff
@@ -65,7 +65,7 @@ export class DocumentFileUploader extends Component {
                 this.notification.add(msg, {
                     title: file,
                     type: "info",
-                    sticky: true,
```

Comment:
I'm not sure about this commit. `sticky` is self-explanatory, well-known by everyone, and it's short. I would keep the props.